astropy/units/quantity_helper/scipy_special.py

```diff
@@ -18,12 +18,46 @@
 
 # ufuncs that require dimensionless input and give dimensionless output.
 dimensionless_to_dimensionless_sps_ufuncs = (
-    'erf', 'erfc', 'erfcx', 'erfi', 'erfinv', 'erfcinv',
-    'gamma', 'gammaln', 'loggamma', 'gammasgn', 'psi', 'rgamma', 'digamma',
-    'wofz', 'dawsn', 'entr', 'exprel', 'expm1', 'log1p', 'exp2', 'exp10',
-    'j0', 'j1', 'y0', 'y1', 'i0', 'i0e', 'i1', 'i1e',
-    'k0', 'k0e', 'k1', 'k1e', 'itj0y0', 'it2j0y0', 'iti0k0', 'it2i0k0',
-    'ndtr', 'ndtri')
+    'erf',
```

Comment:
@mhvk. Maybe we should be storing these in JSON files and having read-only variables here? Users might also benefit from being able to tell the input/output expectations for a function, without having to run the function. These lists determine that.

Reply:
Perhaps a better way to do this is indeed possible, though I think in the end users would hopefully not need this at all...